autoload/neomake/makers/ft/ruby.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#ruby#EnabledMakers()
-    return ['mri', 'rubocop', 'reek']
+    return ['mri', 'rubocop', 'reek', 'rubylint']
```

Comment:
I think we should not return this makers without executable() check;
see #375 
@blueyed 
I think `EnabledMakers()` should only return the effective maker in current os;
for example
```vim
function! neomake#makers#ft#ruby#EnabledMakers()
      let makers = []
      if executable('ruby-lint')
           call add(makers,'ruby-lint')
      endif
      return makers    
endf
```